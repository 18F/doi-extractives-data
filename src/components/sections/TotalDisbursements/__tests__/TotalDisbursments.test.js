import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import TotalDisbursements from '../TotalDisbursements'
import { configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


const mockStore = configureMockStore();
const store = mockStore({});

describe("TotalDisbursements Component", () => {
    it("should render without throwing an error", () => {

        const tree= shallow(
            <Provider store={store}>
              <TotalDisbursements />
            </Provider>
        ).exists()
        expect(toJson(tree)).toMatchSnapshot()
    });
});



