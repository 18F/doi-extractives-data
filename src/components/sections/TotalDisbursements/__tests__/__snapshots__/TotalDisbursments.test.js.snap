// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalDisbursements Component should render without throwing an error 1`] = `undefined`;
